src/components/BPMSlider.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+// eslint-disable-next-line import/extensions
```

Comment:
Why disable eslint here? If the eslint rule does not suit for you, why not just change the rule?